app/views/staff/program_sessions/_form.html.haml

```diff
@@ -9,12 +9,14 @@
         = f.input :state, as: :select, collection: session_states_collection, label: "State",
           include_blank: false, required: true
       = f.input :abstract, maxlength: 605, input_html: { class: 'watched js-maxlength-alert', rows: 5 }
-      -# %section.inline-block
-      -#   = f.label :video_url
-      -#   = f.text_field :video_url, class: "form-control"
-      -# %section.inline-block
-      -#   = f.label :slides_url
-      -#   = f.text_field :slides_url, class: "form-control"
+      %section.inline-block
+        = f.label :video_url
+        = f.text_field :video_url, class: "form-control", maxlength: 100
```

Comment:
I'm curious why there's a max length of 100 here.  Though it seems likely that a url will not exceed 100 characters, I don't know if that will always be true.